content/blog/2020-convince-your-boss.md

```diff
@@ -0,0 +1,44 @@
+Title: Help us, Help you Convince Your Boss  
+Date: 2020-08-28 23:00
+Author: Vipul Gupta (@vipulgupta2048)
+Category: 2020
+Slug: 2020-convince-your-boss 
+Summary: Save your time composing an email to convince your boss for you to attend PyCon India 2020. We got you covered.
+
+There is lots to be gained by attending PyCon India 2020. Unfortunately, it takes time to sit down and compose an email to send to your boss enumerating all of them. Now consider that at least half of our attendees attempt to do that, we would lose so many hours of productivity reinventing the wheel ;) Hence, in the spirit of Python and automation, the PyCon India Content team is here to save the day. Down below, we have composed a "Convince your boss" letter. Of course, feel free to edit the parts you need and remove some you don't. 
+
+## The Letter
+
+Dear `{BOSS}`,
+
+I am excited to brief you about PyCon India 2020, India's premier conference on using and developing the Python programming language that takes place online from October 2-5, 2020. About a 1500+ of the industry's Pythonistas from different fields and all over the world are bound to be in attendance for the 12th edition of this mega event.
```

Comment:
s/industry's Pythonistas/ Pythonistas/

The crowd, though majority from companies, is a balance of all

Reply:
Yeah, makes sense.